Python/Tests/TestAdapterTests/TestExecutorTests.cs

```diff
@@ -209,6 +209,7 @@ public class TestExecutorTests {
                 AssertUtil.ContainsAtLeast(resultNames, expectedResult.TestCase.FullyQualifiedName);
                 var actualResult = recorder.Results.SingleOrDefault(tr => tr.TestCase.FullyQualifiedName == expectedResult.TestCase.FullyQualifiedName);
                 Assert.AreEqual(expectedResult.Outcome, actualResult.Outcome, expectedResult.TestCase.FullyQualifiedName + " had incorrect result");
+                Assert.IsTrue(actualResult.Duration >= TimeSpan.Zero);
```

Comment:
I'm okay with a test that sleeps for 2 seconds and we ensure the duration is greater than 1.5s - it's significantly better than our current state. But if it's relatively easy to do on the Python side then go ahead.